sdk/iot/Azure.ResourceManager.IotFirmwareDefense/api/Azure.ResourceManager.IotFirmwareDefense.net8.0.cs

```diff
@@ -462,57 +653,80 @@ protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer
+        public string Description { get { throw null; } set { } }
+        public string FileName { get { throw null; } set { } }
+        public long? FileSize { get { throw null; } set { } }
+        public string Model { get { throw null; } set { } }
+        public Azure.ResourceManager.IotFirmwareDefense.Models.FirmwareProvisioningState? ProvisioningState { get { throw null; } }
+        public Azure.ResourceManager.IotFirmwareDefense.Models.FirmwareAnalysisStatus? Status { get { throw null; } set { } }
+        public System.Collections.Generic.IList<Azure.ResourceManager.IotFirmwareDefense.Models.FirmwareAnalysisStatusMessage> StatusMessages { get { throw null; } }
+        public string Vendor { get { throw null; } set { } }
+        public string Version { get { throw null; } set { } }
+        protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer, System.ClientModel.Primitives.ModelReaderWriterOptions options) { }
+        Azure.ResourceManager.IotFirmwareDefense.Models.FirmwareProperties System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.IotFirmwareDefense.Models.FirmwareProperties>.Create(ref System.Text.Json.Utf8JsonReader reader, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        void System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.IotFirmwareDefense.Models.FirmwareProperties>.Write(System.Text.Json.Utf8JsonWriter writer, System.ClientModel.Primitives.ModelReaderWriterOptions options) { }
+        Azure.ResourceManager.IotFirmwareDefense.Models.FirmwareProperties System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.IotFirmwareDefense.Models.FirmwareProperties>.Create(System.BinaryData data, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.IotFirmwareDefense.Models.FirmwareProperties>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.IotFirmwareDefense.Models.FirmwareProperties>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+    }
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
     public readonly partial struct FirmwareProvisioningState : System.IEquatable<Azure.ResourceManager.IotFirmwareDefense.Models.FirmwareProvisioningState>
     {
         private readonly object _dummy;
         private readonly int _dummyPrimitive;
         public FirmwareProvisioningState(string value) { throw null; }
-        public static Azure.ResourceManager.IotFirmwareDefense.Models.FirmwareProvisioningState Accepted { get { throw null; } }
+        public static Azure.ResourceManager.IotFirmwareDefense.Models.FirmwareProvisioningState Analyzing { get { throw null; } }
```

Comment:
the `Accepted` value was removed.
Please add this back in a partial struct, like this:
```
namespace Copy.The.Original.Namespace.Here
{
	public readonly partial struct FirmwareProvisioningState
	{
		public static FirmwareProvisioningState Accepted { get; } = new("its original value"); // please find the original value before our change.
	}
}
```